lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.html

```diff
@@ -204,9 +204,9 @@
         <div *ngFor="let group of groupedProperties; let first = first" class="adf-metadata-grouped-properties-container">
             <mat-expansion-panel
                 [attr.data-automation-id]="'adf-metadata-group-' + group.title"
-                [expanded]="currentPanel.panelTitle === group.title && currentPanel.expanded"
-                (opened)="expandPanel(group.title)"
-                (closed)="closePanel(group.title)"
+                [expanded]="group.title === displayAspect || !displayDefaultProperties && first || currentPanel.panelTitle === group.title && group.expanded"
```

Comment:
`group.title === displayAspect` is already checked in `OnInit` and `OnChanges` and then compared with `currentPanel` so I think that adding `!displayDefaultProperties && first` part would be enough

Reply:
right, removed `group.title === displayAspect` as it's already checked